tests/unittests/core/evc/test_conflicts.py

```diff
@@ -372,6 +360,52 @@ def test_try_resolve(self, experiment_name_conflict):
         assert resolution.conflict is experiment_name_conflict
         assert resolution.new_name == new_name
 
+    def test_branch_w_existing_exp(self, existing_exp_conflict):
+        """Test branching when an existing experiment with the new name already exists"""
+        with pytest.raises(ValueError) as exc:
+            existing_exp_conflict.try_resolve("dummy")
+
+        assert "Cannot" in str(exc.value)
+
+    def test_conflict_exp_no_child(self, exp_no_child_conflict):
+        """Verify the version number is incremented when exp has no child."""
+        new_name = "test"
+        assert not exp_no_child_conflict.is_resolved
+        resolution = exp_no_child_conflict.try_resolve(new_name)
+        assert isinstance(resolution, exp_no_child_conflict.ExperimentNameResolution)
+        assert exp_no_child_conflict.is_resolved
+        assert resolution.conflict is exp_no_child_conflict
+        assert resolution.new_version == 2
```

Comment:
should we test that `exp_no_child_conflict.old_config['version'] == 1` to make sure there is indeed an increment?